README.md

```diff
@@ -472,6 +472,11 @@ Some storage classes have limitations requiring the test to skip some parts:
 - `--skip-resize-job` - Skip volume resize job. Use when e.g. `allowVolumeExpansion` is `false`
 - `--skip-migration-job` - Skip the migration job. Use when e.g. `RWX` `accessMode` is not supported
 
+#### Cleanup
+
+Since the test is expected to run until failure, it is designed to keep all allocated resources to allow investogating the failure.
```

Comment:
nit: `s/investogating/investigating/g`

Reply:
Thanks, fixed (BTW we should add a spell check to the linters test)